styles/dark.css

```diff
@@ -18,7 +18,7 @@
   justify-content: center;
   flex-direction: column;
   align-items: center;
-  background-color: rgb(20, 19, 19);
+  background-color: rgb(66, 66, 61);
```

Comment:
I think this color would be more fitting for the light mode. You could pick something darker for the background and make the buttons this color.